### ISSUE TITLE
Add ability to display full day event

### ISSUE THREAD

**Issue by rdewolff** [hoangnm/react-native-week-view#132]
As a user, I wish to see full day events on the top of my day in a dedicated space, to avoid filling too much space in the week view.

This is usually an allDay Boolean in the event object to decide if this event should be displayed differently.

Cf info https://github.com/wmcmahan/react-native-calendar-events#event-fields

![2C093857-A005-46F8-AE5A-29832589E339](https://user-images.githubusercontent.com/130704/131295106-197a3e4d-667e-447f-a7d0-50a81151085a.jpeg)

**Comment by rdewolff** [comment]
What do you guys think about this addition?

**Comment by pdpino** [comment]
I think this feature is broad enough for the library, it would be nice to implement it.
But it would take some time. Some considerations:

* The all-day events behave somewhat different from regular events:
  * cannot be dragged, at least not vertically (upcoming feature)
  * endHour cannot be edited by long-pressing (upcoming feature)
* The dedicated-space height should grow according to the amount of all-day events
* When scrolling horizontally, the dedicated-space height may change in different weeks --> should be smoothly animated (ideally)

**Comment by rdewolff** [comment]
Yeah this is an important feature for any calendar. Any idea Hope we can have this soon 👍

**Comment by TheTushar2696** [comment]
@pdpino  any plans to pick this up in the near future?

**Comment by pdpino** [comment]
Yes sure, I will consider it in the upcoming features

**Comment by rdewolff** [comment]
Looking forward having this too!

**Comment by pdpino** [comment]
I have a first version for this feature:

https://user-images.githubusercontent.com/20805451/188169096-9cd43795-ecf3-41d1-9d7a-a5e5398ea849.mp4

I will send a PR after block-events #220 though (to avoid merge conflicts)
